mirdata/datasets/idmt_smt_audio_effects.py

```diff
@@ -65,21 +65,28 @@
     "full_dataset": download_utils.RemoteFileMetadata(
         filename="IDMT-SMT-AUDIO-EFFECTS.zip",
         url="https://zenodo.org/record/7544032/files/IDMT-SMT-AUDIO-EFFECTS.zip?download=1",
-        checksum="91e845a1b347352993ebd5ba948d5a7c",  # the md5 checksum
-        destination_dir=".",  # relative path for where to unzip the data, or None
-        unpack_directories=[""],
+        checksum="91e845a1b347352993ebd5ba948d5a7c",
     ),
 }
 
 DOWNLOAD_INFO = """
-        This loader will create the following folders in the dataset data_home path:
-            > idmt_smt_audio_effects/
-                > Bass monophon/
-                > Bass monophon2/
-                > Gitarre monophon/
-                > Gitarre monophon2/
-                > Gitarre polyphon/
-                > Gitarre polyphon2/
+    Folder tree:
+    data_home/
```

Comment:
```suggestion
```data_home/
    ├── Bass monophon/
    │   ├── Lists
    │   └── Samples
    ├── Bass monophon2
    ├── Gitarre monophon/
    │   ├── Lists
    │   ├── Samples
    │   └── ReadMe.txt
    ├── Gitarre monophon2
    ├── Gitarre polyphon/
    │   ├── Lists
    │   └── Samples
    ├── Gitarre polyphon2
    └── IDMT-SMT-AUDIO-EFFECTS/
        └── IDMT-SMT-Audio-Effects-Description.pdf

Reply:
I've generated the tree using: https://tree.nathanfriend.io/, I think it's easier to read now.